charts/endeavor/templates/pvc.yaml

```diff
@@ -0,0 +1,35 @@
+{{- if and (not .Values.useStatefulSet) .Values.persistence.enabled (not .Values.persistence.existingClaim) (eq .Values.persistence.type "pvc")}}
```

Comment:
I created this helm chart based on this [example](https://github.com/grafana/helm-charts/blob/main/charts/grafana/templates/pvc.yaml)

I am not sure though if this is the correct approach or using a StatefulSet is the way to go